src/server/pfs/server/api_server.go

```diff
@@ -277,6 +277,23 @@ func (a *apiServer) FinishCommit(ctx context.Context, request *pfs.FinishCommitR
 	return &emptypb.Empty{}, nil
 }
 
+// ForgetCommit implements the protobuf pfs.ForgetCommit RPC
+func (a *apiServer) ForgetCommit(ctx context.Context, req *pfs.ForgetCommitRequest) (*pfs.ForgetCommitResponse, error) {
+	if err := errors.Wrap(a.driver.txnEnv.WithReadContext(ctx, func(ctx context.Context, txnCtx *txncontext.TransactionContext) error {
+		c, err := pfsdb.PickCommit(ctx, req.Commit, txnCtx.SqlTx)
```

Comment:
Instead of repicking the commit here, you can change the function parameters to take a `*pfsdb.Commit`, then you can just call DropFileSetsTx on that.

Reply:
For example:
``` go
func (a *apiServer) ForgetCommitTx(ctx context.Context, txnCtx *txncontext.TransactionContext, commit *pfsdb.Commit) error { ... }
```

Reply:
Got it. So ForgetCommit is removed? We only have ForgetCommitTx?